specification/datalake-analytics/data-plane/Microsoft.DataLakeAnalytics/stable/2016-11-01/job.json

```diff
@@ -1483,7 +1488,12 @@
           "type": "integer",
           "default": 1,
           "format": "int32",
-          "description": "the degree of parallelism used for this job. This must be greater than 0, if set to less than 0 it will default to 1."
+          "description": "the degree of parallelism used for this job. At most one of degreeOfParallelism and degreeOfParallelismPercent should be specified. If none, a default value of 1 will be used."
+        },
+        "degreeOfParallelismPercent": {
+          "type": "number",
+          "format": "double",
+          "description": "the degree of parallelism in percentage used for this job. At most one of degreeOfParallelism and degreeOfParallelismPercent should be specified. If none, a default value of 1 will be used for degreeOfParallelism."
```

Comment:
I don't see JobInformationBasic or any type deriving from it being used as PUT|PATCH|POST request payload. So far they seems used to represent data returned from the service, please correct me if I'm wrong. If so should we mark these properties as readonly in JobInformationBasic model?

Reply:
You are right Anu, I just updated the change, please take another look. Thanks.